src/admin/vote/Results.jsx

```
@@ -2,8 +2,9 @@ import { collection, doc, getDoc, getDocs, setDoc } from "firebase/firestore";
 import { useLoaderData } from "react-router-dom";
 import { auth, db } from "../../firebase";
 
-import { confirm, snackbar } from "mdui";
+import { confirm, prompt, snackbar } from "mdui";
 import React from "react";
+import { max } from "numeric";
```

ESLint

[error] 'max' is defined but never used.

(no-unused-vars)